src/basic_commands/block_size.md

```diff
@@ -28,13 +28,12 @@ The `b` command is used to change the block size:
 The `bf` command is used to change the block size to value specified by a flag. For example, in symbols, the block size of the flag represents the size of the function.
 ```
 [0x00000000]> bf sym.main    ; block size = sizeof(sym.main)
-[0x00000000]> pd @ sym.main  ; disassemble sym.main
+[0x00000000]> pD @ sym.main  ; disassemble sym.main
```

Comment:
maybe explain $F for function size and reference the page explaining all the $ variables documented in ?$?

Reply:
Still haven't reached that chapter yet. Thanks for the hint!